.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: stripe-php CI
+
+on: [push, pull_request]
+
+jobs:
+  php-cs-fixer:
+    name: PHP-CS-Fixer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: PHP-CS-Fixer
+        uses: docker://oskarstark/php-cs-fixer-ga
+        with:
+          args: --diff-format=udiff --dry-run --show-progress=dots --using-cache=no --verbose .
+
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: PHPStan
+        uses: docker://oskarstark/phpstan-ga
+        with:
+          args: analyse -c phpstan.neon lib tests
+
+  tests:
+    name: "Test: PHP ${{ matrix.php-version }} / ${{ matrix.env }}"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        env:
+          - AUTOLOAD=0
+          - AUTOLOAD=1
+        php-version:
+          - "5.6"
+          - "7.0"
+          - "7.1"
+          - "7.2"
+          - "7.3"
+    steps:
+      - uses: actions/checkout@master
```

Comment:
Super minor, but if you're down: I found the spacing between steps that you had in the prototype made this list easier to read.